### PR TITLE
סוגריים

### PR DESCRIPTION
**Pull request by SalehKa**

### CODE REVIEW

File: lisp.tex [hunk 1]
```diff
@@ -25,14 +25,14 @@
 }{%
   \csname endfilecontents*\endcsname%
   \LTR
-    \lstinputlisting[language=kernel,style=display,backgroundcolor=\color{olive!10}]{\fileName}%
+  \lstinputlisting[language=kernel,style=display,backgroundcolor=\color{olive!10}]{\fileName}%
```
Comment (gur111) [inline lisp.tex:28]: Why remove the indentation?

File: lisp.tex [hunk 2]
```diff
@@ -25,14 +25,14 @@
...
   \endLTR
   \openin \tempFile=\fileName
   \begingroup\endlinechar=-1
   \loop\unless\ifeof \tempFile
-    \read\tempFile to\fileline % Read one line and store it into \fileline
-    \immediate\write \kernelFile
-    {\unexpanded\expandafter{\fileline}}
+  \read\tempFile to\fileline % Read one line and store it into \fileline
```
Comment (gur111) [inline lisp.tex:33]: Bring back indentation

File: lisp.tex [hunk 3]
```diff
@@ -46,15 +46,15 @@
...
   \newread \tempFile % open the file to read from
   \openin \tempFile=\fileName
   \begingroup\endlinechar=-1
   \loop\unless\ifeof \tempFile
-    \read\tempFile to\fileline % Read one line and store it into \fileline
-    \immediate\write \libraryFile
-    {\unexpanded\expandafter{\fileline}} % print the content to copy.txt
+  \read\tempFile to\fileline % Read one line and store it into \fileline
+  \immediate\write \libraryFile
+  {\unexpanded\expandafter{\fileline}} % print the content to copy.txt
```
Comment (gur111) [inline lisp.tex:57]: indentation again

File: lisp.tex [hunk 4]
```diff
@@ -46,15 +46,15 @@
 }{%
   \csname endfilecontents*\endcsname%
   \LTR
-    \lstinputlisting[language=kernel,style=display,backgroundcolor=\color{orange!20}]{\fileName}%
+  \lstinputlisting[language=kernel,style=display,backgroundcolor=\color{orange!20}]{\fileName}%
```
Comment (gur111) [inline lisp.tex:49]: indent

File: lisp.tex [hunk 5]
```diff
@@ -63,15 +63,11 @@
 %
 § מבוא
 
-שפת LISP (ראשי תיבות של \E|LIst PRocessing|), היתה, יחד עם שפת Fortran (ראשי
+שפת LISP (כקיצור של \E|LIst ProceSsing|), היתה, יחד עם שפת Fortran (ראשי
```
Comment (gur111) [inline lisp.tex:66]: Maybe `LISt Processing` but it surely isn't `LIPS`

**Review by gur111 — CHANGES_REQUESTED** [review]
Please resolve all conflicts by merging with master.
```
git checkout master
git pull
git checkout -
git merge master
```
And submit for review again

**Comment by yossigil** [comment]
I am sorry, but the dev version has converged, and there are still conflicts. Can you please create a new PR with respect to the current dev version.

**Pull request closed without merging**